### PR TITLE
Turn on X-Axis braking

### PR DESCRIPTION
**Pull request by Jeff-Ciesielski**
@mdicicco: I'd like to try this in response to the email re: lost x-axis counts.

Thoughts?

### CODE REVIEW

**Comment by mdicicco** [comment]
the last domino falls... we're now braking on every axis?

File: cpu_map_keyme2560.h [hunk 1]
```diff
@@ -62,7 +62,7 @@
 #define STEPPERS_DISABLE_MASK ((1<<X_DISABLE_BIT)|(1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1<<C_DISABLE_BIT))   // All disable bits
 
 //Keep gripper, carousel, and Y engaged longer
```
Comment (zsalzbank) [inline cpu_map_keyme2560.h]: Update this too?

File: cpu_map_keyme2560.h [hunk 2]
```diff
@@ -61,8 +61,8 @@
 #define C_DISABLE_BIT 5  // J5 /Atmega Pin 68
 #define STEPPERS_DISABLE_MASK ((1<<X_DISABLE_BIT)|(1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1<<C_DISABLE_BIT))   // All disable bits
 
-//Keep gripper, carousel, and Y engaged longer
-#define STEPPERS_LONG_LOCK_MASK  ((1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1 << C_DISABLE_BIT))
+//Keep all axis engaged
+#define STEPPERS_LONG_LOCK_MASK  ((1<<X_DISABLE_BIT)|(1<<Y_DISABLE_BIT)|(1<<Z_DISABLE_BIT)|(1 << C_DISABLE_BIT))
```
Comment (mdicicco) [inline cpu_map_keyme2560.h]: you change the style around the "<<" operator in the middle of the line

Comment (Jeff-Ciesielski) [reply cpu_map_keyme2560.h]: Oops, good catch, will fix.

**Comment by mdicicco** [comment]
:+1:

**Pull request merged**